bindings/src/main/java/io/pravega/storage/filesystem/FileSystemStorageConfig.java

```diff
@@ -24,6 +24,7 @@
     //region Config Names
 
     public static final Property<String> ROOT = Property.named("root", "/fs/");
+    public static final Property<Boolean> REPLACE_ENABLED = Property.named("replace.enabled", false);
```

Comment:
> is the full property name filesystem.replace.enable

Yes.

> are we omitting this configuration in config.properties intentionally?

Yes. This is to support a stop-gap fix for RollingStorage which will not be needed once SLTS will replace it. I do not want to advertise it to potential users and stir confusion when it will not be there anymore.

Reply:
> it should be enable rather than enabled.

Fixed.